internal/core/errors/errors.go

```diff
@@ -0,0 +1,61 @@
+package errors

```

Comment:
why do you implemented std erorrs?

Reply:
to use custom errors as const

Reply:
error is interface that's why you cannot define it's as a const